packages/web/src/components/gcds-pagination/utils/render.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Function to constuct href attribute from url object
+ * Also looks for ::offset and ::match to increment query string values
+ */
+function constructHref(el, page: number, end?: "next" | "previous" | null) {
+  let fragment = "";
+  let qs = "";
+
+  if (el.url) {
+    let count = 0;
+    for (const key in el.url.queryStrings) {
```

Comment:
Sorry very nit-picky again, but for consistency: please add a space before and after each if statement, for loops, etc.